tests/proofs/dexter/dexter-spec.md

```diff
@@ -267,3 +267,119 @@ If any of the conditions are not satisfied, the call fails.
 ```k
 endmodule
 ```
+
+## Update Token Pool Internal
+
+```k
+module DEXTER-UPDATETOKENPOOLINTERNAL-SPEC
+  imports DEXTER-VERIFICATION
+```
```

Comment:
Let's make sure we state that we need to be sure (on paper) that the list of claims here is exhaustive.